src/client/nativeCommunication.ts

```diff
@@ -194,12 +201,12 @@ function reportNativeElementPositionChanges(): void {
     observer.observe(targetNode, config);
 
     try {
-        document.fonts.ready.then(() => {
-            commercialClient.updateAdverts(getAdSlots());
-            videoClient.updateVideos(getVideoSlots());
+        void document.fonts.ready.then(() => {
+            void commercialClient.updateAdverts(getAdSlots());
+            void videoClient.updateVideos(getVideoSlots());
         });
     } catch (e) {
-        logger.error(`font loading API not supported: ${e}`)
+        logger.error(`font loading API not supported: ${errorToString(e, 'unknown reason')}`);
```

Comment:
It could do. I'm not very keen on default parameters because I feel they complicate the function signature. If you think it's necessary to simplify this I'd prefer a second function:

```ts
const errorToStringWithFallback = (fallback: string) => (e: unknown): string => ...

const errorToString = errorToStringWithFallback('unknown reason');
```